libs/flowbite-angular/alert/alert.component.ts

```
@@ -13,17 +12,87 @@ import {
+);
+
+export const FLOWBITE_ALERT_ADDITIONAL_CONTENT_DEFAULT_VALUE =
+  new InjectionToken<TemplateRef<unknown> | null>(
+    'FLOWBITE_ALERT_ADDITIONAL_CONTENT_DEFAULT_VALUE'
+  );
+
+export const FLOWBITE_ALERT_IS_DISMISSABLE_DEFAULT_VALUE = new InjectionToken<boolean>(
+  'FLOWBITE_ALERT_IS_DISMISSABLE_DEFAULT_VALUE'
+);
+
+export const FLOWBITE_ALERT_ON_DISMISS_DEFAULT_VALUE = new InjectionToken<() => void | undefined>(
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)